### PR TITLE
Update requirements-network.md

### PR DESCRIPTION
**Pull request by saudmish**
Fixed a broken link (I used the wrong brackets) and made language slightly more explicit.

### CODE REVIEW

**Comment by PRMerger4** [comment]
@saudmish : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ErikjeMS** [comment]
@saudmish My review above. Also, in the future, please make your changes in the memdocs-pr repo. That's where we prefer MS employees to submit changes.

File: windows-365/enterprise/requirements-network.md [hunk 1]
```diff
@@ -94,11 +94,11 @@ All endpoints connect over port 443.
...
+To make it easier to configure network security controls, use Azure Networking service tags, the Azure Virtual Desktop service tag, to identity those endpoints for direct routing using an [Azure Networking User Defined Route (UDR)](/azure/virtual-network/virtual-networks-udr-overview). A UDR will result in direct routing between your virtual network and the RDP broker for lowest latency. For more information about Azure Service Tags with Azzur Virtual Desktop endpoints, see [Azure service tags overview](/azure/virtual-desktop/network-connectivity).
 
-Changing the network routes of a Cloud PC (at the network layer or at the Cloud PC layer (e.g. VPN)), might break the connection between the Cloud PC and the Azure Virtual Desktop RDP broker. If so, the end user will be disconnected from their Cloud PC until a connection be established.
+Changing the network routes of a Cloud PC (at the network layer or at the Cloud PC layer (e.g. VPN)), might break the connection between the Cloud PC and the Azure Virtual Desktop RDP broker. If so, the end user will be disconnected from their Cloud PC until a connection be re-established.
```
[the remark this answers is not in the document]
Comment (saudmish) [reply windows-365/enterprise/requirements-network.md:101]: Looks good!

**Review by saudmish — COMMENTED** [review]
Good feedback, we should grab 15 minutes as I'm struggling to get my point across in writing. :)

**Comment by PRMerger3** [comment]
@ErikjeMS : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by ErikjeMS — APPROVED** [review]

**Comment by ErikjeMS** [comment]
#sign-off

**Comment by PRMerger10** [comment]
@ErikjeMS : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by ErikjeMS — APPROVED** [review]

**Comment by ErikjeMS** [comment]
#sign-off

**Pull request merged**